prisma/schema.prisma

```diff
@@ -57,6 +58,6 @@ model subscriptions {
   topic          String?
   createdAt      DateTime? @default(now()) @db.Timestamptz(6)
   updatedAt      DateTime? @default(now()) @db.Timestamptz(6)
```

Comment:
Be nice to make these fields non optional in the future.

Reply:
I made the date fields non-nullable (createdAt, updatedAt). The topic is a bit messy, as it is null for the subscription type cabins and only needed for the type activities.